workspace/assets/css/site/util-spacing.less

```diff
@@ -0,0 +1,446 @@
+//=========================
+// = UTILITIES FOR SPACING
+//=========================
+
+
+//====== GRIDS ============
+
+// #generate-strict-guttered-grid(1);
+// #generate-strict-guttered-grid(2);
+// #generate-strict-guttered-grid(3);
+// #generate-strict-guttered-grid(4);
+// #generate-strict-guttered-grid(5);
+
+
+//====== SERIES RHYTHM ============
+
+.spaced-vertical-small {
+	> * + * {
+		margin-top: @spacing-small;
+	}
+}
+
+.spaced-vertical-smallest {
+	> * + * {
+		margin-top: @spacing-smallest;
+	}
+}
+
+//====== MAX-WIDTHS ============
+
+.column-to-sm-row {
+	.flex-flow-column-wrap;
+	
+	#mq-min-width(@screen-sm, {
+		.flex-flow-row-wrap;
+	});
+}
+
+.column-to-xs-row {
+	.flex-flow-column-wrap;
+	
+	#mq-min-width(@screen-xs, {
+		.flex-flow-row-wrap;
+	});
+}
+
+
+//====== MAX-WIDTHS ============
+
+.max-width-smaller {
+	max-width: @spacing-smaller * 10;
+}
+
+.max-width-small {
+	max-width: @spacing-small * 10;
+}
+
+.max-width-mid {
+	max-width: @spacing-mid * 10;
+}
+
+.max-width-large {
+	max-width: @spacing-large * 10;
+}
+
+.max-width-larger {
+	max-width: @spacing-larger * 10;
+}
+
+.max-width-largest {
+	max-width: @spacing-larger * 10;
+}
+
+
+.max-width-huge {
+	max-width: @spacing-huge * 10;
+}
+
+// create equal viewport margins on small screens
+[class*="max-width"] {

```

Comment:
Very slow selector and may cause unpredictable behavior. Max width can be used for other than viewport